index.html

```diff
@@ -1,57 +1,213 @@
-            <a href="#" class="social-media-icon">
-              <img src="./icons/github.png" alt="Github"/>
-            </a>
-          </li>
-          <li>
-            <a href="#" class="social-media-icon">
-              <img src="./icons/Linkedin.png" alt="Linkedin"/>
-            </a>
-          </li>
-          <li>
-            <a href="#" class="social-media-icon">
-              <img src="./icons/angel.png" alt="Angel"/>
-            </a>
-          </li>
-          <li>
-            <a href="#" class="social-media-icon">
-              <img src="./icons/twitter.png" alt="Twitter"/>
-            </a>
-          </li>
-          <li>
-            <a href="#"  class="social-media-medium-icon">
-              <img class="image-correction" src="./icons/medium.png" alt="Medium"/>
-            </a>
-          </li>
-        </ul>
-      </section>
-    </section>
-  </body>
-</html> 
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Adaobi Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+    <nav class="container">
+      <a href="#" class="navbar-logo">Adaobi Okoye</a>
+      <a href="#" class="navbar-menu">
+        <img src="/icons/Union.png" alt="Menu"/>
+      </a>
+    </nav>
+    </header>
+    <header class="header-box">
+          <section class="header-content">
+        <h1 class="header-title">Hey There. I'm Adaobi</h1>
+        <h2 class="header-subtitle">I’m a Software Developer</h2>
+        <p class="header-text-description">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/icons/github.png" alt="Github"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="icons/Linkedin.png" alt="Linkedin"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/icons/angel.png" alt="Angel"/>
+            </a>
+          </li>
+          <li>
+            <a href="#" class="social-media-icon">
+              <img src="/icons/twitter.png" alt="Twitter"/>
+            </a>
+          </li>
+          <li>
+            <a href="#"  class="social-media-medium-icon">
+              <img class="image-correction" src="/icons/medium.png" alt="Medium"/>
+            </a>
+          </li>
+        </ul>
+    </section>
+  </header>
+   <main>
+      <section class="new-container">
+        <div class="work-container">
+          <h1 class="recent-work">My Recent Works</h1>
```

Comment:
- [ ] Kindly change the `<h1>` tag to any of `<h2>, <h3>, ..., <h6>` tag.
`<h1>` tags tell search engine bots and web users what a page is about. A webpage's `<h1>` is the most important heading and should accurately summarize the contents of that page.
You can read more [here](https://www.semrush.com/blog/h1-tag/#:~:text=H1%20tags%20tell%20search%20engine,the%20contents%20of%20that%20page.).